challenge-05/challenge-05.js

```diff
@@ -20,18 +25,28 @@ segundo, um número. A função deve retornar um índice do array que foi passad
 no primeiro parâmetro. O índice a ser retornado, deve ser o número passado no
 segundo parâmetro.
 */
-// ?
+
+function myFunction(array, numArray){
+	return array [numArray];
```

Comment:
-1
Sempre que for chamar um índice de array, pode colar o `[` no nome do array, ok?